<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1420</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Online Public Address System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="form_src.qrc">
    <normaloff>:/icon/pcr-icon-on.png</normaloff>:/icon/pcr-icon-on.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1420</width>
      <height>751</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1420</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Seting"/>
    <addaction name="action_Monitor"/>
    <addaction name="separator"/>
    <addaction name="action_Keluar"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="separator"/>
    <addaction name="action_Penggunaan"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>58</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>90</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QPushButton" name="pbSetting">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Setting</string>
     </property>
     <property name="whatsThis">
      <string>Setting</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icon/settings.jpg</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pbMon">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Control</string>
     </property>
     <property name="whatsThis">
      <string>Control</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icon/mon.jpg</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pbHelp">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Help</string>
     </property>
     <property name="whatsThis">
      <string>Help</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>:/icon/help.jpg</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdJam">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>2</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
     <property name="intValue" stdset="0">
      <number>23</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdMenit">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>0</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
     <property name="intValue" stdset="0">
      <number>23</number>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdDetik">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>0</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="digitCount">
      <number>2</number>
     </property>
     <property name="intValue" stdset="0">
      <number>23</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>325</x>
       <y>10</y>
       <width>21</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>10</y>
       <width>21</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>142</x>
       <y>0</y>
       <width>16</width>
       <height>71</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>0</y>
       <width>16</width>
       <height>71</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lokasi">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>-1</y>
       <width>901</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Condensed</family>
       <pointsize>54</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Petra Campus Radio</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>0</y>
       <width>16</width>
       <height>71</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="action_About">
   <property name="text">
    <string>&amp;About (F12)</string>
   </property>
  </action>
  <action name="action_Penggunaan">
   <property name="text">
    <string>&amp;Manual (F1)</string>
   </property>
  </action>
  <action name="action_Seting">
   <property name="text">
    <string>&amp;Setting (F2)</string>
   </property>
  </action>
  <action name="action_Monitor">
   <property name="text">
    <string>&amp;Monitor (F3)</string>
   </property>
  </action>
  <action name="action_Keluar">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="form_src.qrc"/>
 </resources>
 <connections/>
</ui>
